### ISSUE TITLE
Typo in prolog section, kr summary

### ISSUE THREAD

**Issue by RiccardoRiglietti** [nicofirst1/AI_notes#1]
```
5 Prolog
Prolog is the major logic-based programming language 98, it uses a set of definite clauses written in a notation
somewhat different from FOL.
For example:
American(x) ∧ (y) ∧ Sells(x, y, z) ∧ Hostile(z) ⇒ Criminal(x)
becomes
criminal(X) : − american(X), kinderEgg(Y ), sells(X, Y, Z), hostile(Z)

```

There is (y) that should be kinderEgg(y).

**Comment by nicofirst1** [comment]
Can you tell me which line you are referring to?

**Comment by RiccardoRiglietti** [comment]
https://github.com/nicofirst1/AI_notes/blob/master/Knowledge%20Representation/kr_summary.pdf

**Comment by RiccardoRiglietti** [comment]
American(x) ∧ (y) ∧ Sells(x, y, z) ∧ Hostile(z) ⇒ Criminal(x)

**Comment by nicofirst1** [comment]
Thanks, fixed in 158b7de7a567f3db4e5627f0ab7ea7ab30e31a5d
